client/app/scripts/superdesk-publish/views/output-channels.html

```diff
@@ -29,73 +29,93 @@ <h3 translate ng-hide="outputChannel._id" translate>Add New Output Channel</h3>
+                            </select>
+                        </div>
 
-                <div class="field">
-                    <label>
-                        <span translate>Subscribers</span>
-                    </label>
-                    <ul class="pills-list clearfix">
-                        <li class="pull-left" ng-repeat="subscriber in outputChannel.destinations">
-                            <div class="name right-pad">{{subscriberLookup[subscriber].name}}</div>
-                            <div class="actions">
-                                <button type="button" ng-click="removeSubscriber(subscriber)" title="{{:: 'Remove subscriber' | translate }}"><i class="icon-trash"></i></button>
+                        <div class="field">
+                            <label>
+                                <span translate>Subscribers</span>
+                            </label>
+                            <ul class="pills-list clearfix">
+                                <li class="pull-left" ng-repeat="subscriber in outputChannel.destinations">
+                                    <div class="name right-pad">{{subscriberLookup[subscriber].name}}</div>
+                                    <div class="actions">
+                                        <button type="button" ng-click="removeSubscriber(subscriber)" title="{{:: 'Remove subscriber' | translate }}"><i class="icon-trash"></i></button>
+                                    </div>
+                                </li>
+                            </ul>
+                            <select ng-show="newSubscriber._id" class="input" ng-model="newSubscriber._id" ng-change="saveNewSubscriber()">
+                                <option ng-repeat="s in subscribers._items" ng-if="!isIncluded(s)" value="{{s._id}}" translate>{{:: s.name}}</option>
+                            </select>
+                            <button type="button" class="btn btn-info" ng-click="cancelNewSubscriber()" ng-show="newSubscriber._id">
+                                <span translate>Cancel</span>
+                            </button>
+                            <button type="button" class="btn btn-info" ng-click="addNewSubscriber()" ng-hide="newSubscriber._id">
+                                <i class="icon-plus-sign icon-white"></i>
+                                <span translate>Add</span>
+                            </button>
+                        </div>
+                    </tab>
+                    
+
+                    <tab heading="Critical Errors" class="TabContent">
+                        <div class="content privileges-settings">
+                            <div class="horizontal">
+                                <div class="roles-list">
+                                    <div ng-repeat="(key, val) in all_errors | orderBy:name" class="field">
+                                        <label>
+                                            {{key}}: {{val}}
```

Comment:
Are these two values fixed (i.e. remain unchanged)? If so, we should use one-time bindings for them.